src/commonTest/kotlin/kotools/types/collection/NotEmptyMapTest.kt

```diff
@@ -1,15 +1,77 @@
 package kotools.types.collection
 
-import kotlinx.serialization.*
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.KSerializer
+import kotlinx.serialization.SerializationException
 import kotlinx.serialization.builtins.MapSerializer
 import kotlinx.serialization.builtins.serializer
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
 import kotlinx.serialization.json.Json
 import kotools.types.contentShouldEqual
 import kotools.types.shouldHaveAMessage
 import kotlin.random.Random
 import kotlin.test.*
 
 class NotEmptyMapTest {
+    @Test
+    fun notEmptyMapOf_should_pass() {
+        val head: Pair<String, Int> = "a" to Random.nextInt()
+        val tail: Array<Pair<String, Int>> = arrayOf(
+            "b" to Random.nextInt(),
+            "c" to Random.nextInt()
+        )
+        assertEquals(
+            actual = notEmptyMapOf(head, *tail).toMap(),
+            expected = mapOf(head, *tail)
+        )
+    }
```

Comment:
Refactor this test for improving its readability.

```suggestion
    @Test
    fun notEmptyMapOf_should_pass() {
        val head: Pair<Char, Int> = 'a' to Random.nextInt()
        val tail: Array<Pair<Char, Int>> = arrayOf(
            'b' to Random.nextInt(),
            'c' to Random.nextInt()
        )
        val result: NotEmptyMap<Char, Int> = notEmptyMapOf(head, *tail)
        val expected: Map<Char, Int> = mapOf(head, *tail)
        result.entries.toSet() contentShouldEqual expected.entries
    }
```